src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -289,7 +290,38 @@ public ExamResource(UserService userService, CourseService courseService, ExamRe
         }
 
         List<TutorParticipation> tutorParticipations = tutorParticipationService.findAllByCourseAndTutor(course, user);
-        tutorDashboardService.prepareExercisesForTutorDashboard(exercises, tutorParticipations);
+        tutorDashboardService.prepareExercisesForTutorDashboard(exercises, tutorParticipations, true);
+
+        return ResponseEntity.ok(exam);
+    }
+
+    /**
+     * GET /courses/:courseId/exams/:examId:for-exam-tutor-test-run-dashboard
+     *
+     * @param courseId the id of the course to retrieve
+     * @param examId the id of the exam that contains the exercises
+     * @return data about a exam test run including all exercises, plus some data for the tutor as tutor status for assessment
+     */
+    @GetMapping("/courses/{courseId}/exams/{examId}/for-exam-tutor-test-run-dashboard")
+    @PreAuthorize("hasAnyRole('INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<Exam> getExamForTutorTestRunDashboard(@PathVariable long courseId, @PathVariable long examId) {
```

Comment:
Do you need this request? I don't see any additional test run logic. You could probably reuse `getExamForTutorDashboard` but I might be wrong

Reply:
There are differences, as only instructors can access this endpoint. We also do not check against the end date of the exam. For the normal dashboard, we throw a forbidden error, as tutors and instructors are not supposed to assess submissions before the exam is over. 
Additionally, we do not prepare the status statistics for this call, as that is not displayed for the test run assessment dashboard.